design-library/src/components/BccLinkItem/BccLinkItem.vue

```diff
@@ -0,0 +1,16 @@
+<script setup lang="ts">
+import { ArrowForwardIcon } from "@bcc-code/icons-vue";
+type Props = {
+  isLast?: boolean;
+};
+defineProps<Props>();
+</script>
+
+<template>
+  <div class="bcc-link-item border-secondary group" :class="{ 'border-none': isLast }">
+    <label class="bcc-link-item-label"><slot></slot></label>
+    <div class="bcc-link-item-arrow group-hover:pr-0">
+      <ArrowForwardIcon class="h-4 w-4" />
```

Comment:
Some Tailwind classes are used here, they should be moved to CSS classes. I agree it can be a bit cumbersome but it's the choice we made for the library and we should make it as easy as possible to render components with only provided classes.

Reply:
These group classes throw an error if they are added as classes.